elements/rh-accordion/rh-accordion.ts

```diff
@@ -76,7 +76,18 @@ export class RhAccordion extends LitElement {
   }
 
   /**
-   * Sets accordion header's accents position to inline or bottom
+   * Positions the accent slot content relative to the panel's title.
+   * The accent slot contains secondary information such as tags, badges, or other small elements.
+   *
+   * - `inline` (default) - Accents appear on the same line as the title
+   * - `bottom` - Accents appear below the title
+   *
+   * ## Usage guidelines
```

Comment:
Appears to be doubling up on the `accents` attribute rows. One `undefined` and the other with this text. Not sure what's happening.

Also, I'm not sure that we'd want this much content and/or a heading inside a table cell. Perhaps we need to different format?

<img width="936" height="655" alt="Image" src="https://github.com/user-attachments/assets/79024372-13f3-4785-a2db-71d210a2462a" />